refactor/client_side/example_dispatch.py

```diff
@@ -55,9 +56,9 @@ def workflow(a):
 print(result)
 
 
-print("Now using dispatch_sync: ")
+# print("Now using dispatch_sync: ")
 
-print(ct.dispatch_sync(workflow)(2))
+# print(ct.dispatch_sync(workflow)(2))
 # time.sleep(3)
```

Comment:
comments should be just a comment instead of a commented print statement

Reply:
Yeah, they go to the supervisord logs, since it basically captures all the stdout and sends it to a file.

Reply:
Fixed this comment.